src/CssInlinerFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier;
```

Comment:
Please let's move this to a dedicated namespace, e.g., `Pelago\Emogrifier\Factories`. (Suggestions and feedback concerning the namespace name welcome.)